### ISSUE TITLE
xish: documentation typo

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5401]
|<img alt="lovebug356's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| lovebug356 reported|
|-|-|
|Trac ID|trac#5401|
|Type|enhancement|
|Created|2011-11-21 14:15:16Z|
|Branch|https://github.com/twisted/twisted/tree/xish-typo-5401|

There is a small typo in the documentation about the event dispatcher in xish.
http://twistedmatrix.com/documents/current/api/twisted.words.xish.utility.EventDispatcher.html

Attachments:

* [xish_docu_typo.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8b8/8b85a698f4dfaf9bddc347de88f9888b69cdaffa/ae09ca83cdc2587a89e1533201d93617f72d9b1e.diff) (585 bytes) - added by lovebug356 on 2011-11-21 14:16:14Z - xish_docu_typo.diff

<details><summary>Searchable metadata</summary>

```
trac-id__5401 5401
type__enhancement enhancement
reporter__lovebug356 lovebug356
priority__low low
milestone__ 
branch__branches_xish_typo_5401 branches/xish-typo-5401
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__words words
keywords__documentation documentation
time__1321884916000000 1321884916000000
changetime__1322167119000000 1322167119000000
version__None None
owner__jesstess jesstess
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Thanks.  Please add the "review" keyword when there is a patch ready to be reviewed.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess commented|
|-|-|

(In [33227]) Branching to 'xish-typo-5401'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess commented|
|-|-|

(In [33228]) Apply patch xish_docu_typo.diff by lovebug356.

refs [#5401](https://github.com/twisted/twisted/issues/5401)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess set status to `closed`|
|-|-|

(In [33230]) Merge xish-typo-5401

Author: lovebug356
Reviewer: jesstess
Fixes: [#5401](https://github.com/twisted/twisted/issues/5401)

Fix typos in xish.utility docstrings.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess commented|
|-|-|

Thanks for the patch, lovebug356! It looks like this is your first Twisted submission, so welcome aboard and let me or #twisted know if you have any questions about the patch submission or review process.

Next time, please attach a diff generated from the root of your Twisted checkout (more details at http://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch)

Thanks again!
